src/NextGen/DonationForm/Repositories/DonationFormRepository.php

```diff
@@ -382,6 +382,7 @@ public function getTotalRevenue(int $formId): int
     }
 
     /**
+     * @unreleased append formId to first section instead of last with multistep in mind.
```

Comment:
In order to validate the form, we need to dig into the form's schema - so that `formId` needs to be present in every step.  Adding it to the first section resolves this.